src/audio_pixel.h

```diff
@@ -1,81 +1,133 @@
 #define JSON_USE_IMPLICIT_CONVERSIONS 0
-#include "include/json/json.hpp"
 
+#ifdef _WIN32
+#define NOMINMAX
+#define WIN32_LEAN_AND_MEAN
+#endif
+
+#include <format>
```

Comment:
could we move some of these includes to the .cpp file?

Reply:
yes i will